staff/irene-roldan/bash/README.md

```diff
@@ -0,0 +1,92 @@
+# COMMANDS BASH
+
+## pwd - actual folder
+
+```sh
+/c/Users/Irene/workspace/isdi-bootcamp-202402
+```
+
+## ls - show folders inside another folder
+
+```sh
+$ ls
+README.md  staff/
+
+$ ls -1 - allows you to list files in a single column format
+README.md
+staff/
+
+$ ls -a - show hidden files
+
+./  ../  .git/  README.md  staff/
+```
+
+## l - link
+
+## d - directory
+
+## mkdir - create a new folder
+
+```sh
+$ mkdir workspace
+```
+
+## cd - change the directory
+
+```sh
+cd staff/irene-roldan/playground
+```
+
+## mv - (move file to another name) ex mv hola.mundo hola.world
+
+```sh
+$mv hello.world folder-a/folder-b/folder-c
+```
+
+## touch - create text files
+
+```sh
+$ touch readme
+```
+
+## code - open visual studio
+
+## ..q - allow continue writing
+
+```sh
+(END)..q
+```
+
+## rm -rf - delete directly the folder and all the content
+
+```sh
+$ rm -rf workspace/helloworld
+```
+
```

Comment:
Git commands should not be included in this file. If you want, you can create another branch with your git-cheat-sheet file🧐